src/main/java/seedu/storage/LoginStorage.java

```diff
@@ -20,8 +22,20 @@ static void load() {
                 Storage.password = Integer.parseInt(passwordHash);
                 return;
             }
+            createNewUser();
```

Comment:
Not completely sure about this but maybe change it to an if else statement instead of an if statement with return.